Libs/MRML/DisplayableManager/vtkMRMLCameraWidget.cxx

```diff
@@ -322,6 +326,10 @@ bool vtkMRMLCameraWidget::ProcessInteractionEvent(vtkMRMLInteractionEventData* e
       this->SaveStateForUndo();
       this->Dolly(0.8);
       break;
+    case WidgetEventToggleStabilizer:
```

Comment:
"Stabilizer" could mean many things. For me, it means camera stabilization (removing shaking), and we already have a transform stabilizer module in SlicerIGT for this purpose.

It should be called something like `WidgetEventToggleCameraTiltLock`.